.github/ISSUE_TEMPLATE/config.yml

```
@@ -0,0 +1,11 @@
+blank_issues_enabled: true
+contact_links:
+  - name: MDN Community Discussion
+    url: https://github.com/orgs/mdn/discussions
+    about: Will the issue require some discussion to define actionable tasks? Consider starting a discussion first.
```

We need to prevent users from avoiding opening discussions when a direct issue is more appropriate. I suggest making the scope clearer:

```suggestion
    about: Does the issue involve a lot of pages, or are you not sure how it can be split into actionable tasks? Consider starting a discussion first.
```

---

Wouldn't https://github.com/mdn/content/discussions be used for those discussions instead of the org-wide Discussions?

---

@nschonni The content repo discussion is not actively used anymore. Any content discussion also happens in the org discussion.

---

Gotcha. Might be worth trying to transfer the issues to the org level and turning them off if possible.

---

> Gotcha. Might be worth trying to transfer the issues to the org level and turning them off if possible.

Yup, we are in the process of doing that :+1: